client/gutenboarding/gutenberg-styles/_animations.scss

```diff
@@ -1,3 +1,5 @@
+/* stylelint-disable */
```

Comment:
Do we need all these?

Reply:
Yeah, `npm run lint:css` complains otherwise. :-( I should've added them when I added the files.

Anyhoo, looking forward to get rid of these files and import from new `@wordpress/base-styles` instead! 🎉